### PR TITLE
316 - Add GA custom event for Compendia to DOM elements

### PR DESCRIPTION
**Pull request by nozomione**
## Issue Number
Closing #316

## Purpose/Implementation Notes
 - Added custom events for compendia to DOM elements using the GA custom events script
 - Verified that custom events are firing correctly in the `development` environment

To see the compendia custom GA4 reports, please login to CCDL Analytics account, and navigate to the `development` property [`Refine.bio Web - Dev`](https://analytics.google.com/analytics/web/#/p422130981/reports/reportinghub?params=_u..nav%3Dmaui&collectionId=6630624882) (issue https://github.com/AlexsLemonade/refinebio-web/issues/310).

## Types of changes
- New feature (non-breaking change which adds functionality)

## Checklist
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by nozomione** [comment]
This PR is ready for your review, thank you David!

File: src/analytics/gtag.js [hunk 1]
```diff
@@ -31,13 +32,13 @@ const trackCompendiaDownload = (compendia) => {
...
   const payload = {}
-  payload.rnaseq_organism = compendia.primary_organism_name
+  payload.rnaseq_organism = getFormattedOrganismName(compendia.organism)
   event(`compendia_rnaseq_download`, payload)
 }
 // for Normalized compendia
 const trackDownloadNormalizedCompendia = (compendia) => {
   const payload = {}
-  payload.normalized_organism = compendia.primary_organism_name
+  payload.normalized_organism = getFormattedOrganismName(compendia.organism)
```
Comment (davidsmejia) [inline src/analytics/gtag.js:41]: You can just call `formatString` here directly

File: src/components/Compendia/Download.js [hunk 2]
```diff
@@ -105,6 +106,10 @@ export const Download = ({ type }) => {
 
   const handleFileDownload = async (id) => {
     const response = await downloadCompendia(id)
+    gtag.compendiaDownload(
+      type.toLowerCase(),
+      formatString(selectedOrganism.primary_organism_name)
+    )
```
[the remark this answers is not in the document]
Comment (nozomione) [reply src/components/Compendia/Download.js]: Since we're now passing the `compendia`, we can utilize `compendia.quant_sf_only` 👍 !

**Comment by nozomione** [comment]
I've applied the feedback and this PR is ready for your review. Thank you David!

**Review by davidsmejia — APPROVED** [review]
LGTM

**Pull request merged**